gazebo_ros/src/gazebo_ros_api_plugin.cpp

```diff
@@ -193,7 +196,20 @@ void GazeboRosApiPlugin::loadGazeboRosApiPlugin(std::string world_name)
   pub_model_states_connection_count_ = 0;
 
   /// \brief advertise all services
-  advertiseServices();
+  if (enable_ros_network_)
+    advertiseServices();
+
+  // Manage clock for simulated ros time
+  pub_clock_ = nh_->advertise<rosgraph_msgs::Clock>("/clock",10);
```

Comment:
/clock is already advertised within advertiseServices(), perhaps it should be removed there.